### ISSUE TITLE
Console output from SFTP post upload command

### ISSUE THREAD

**Issue by enly1** [mkloubert/vs-deploy#94]
Hi,

Not sure if the underlying protocol supports it, but would be great if as part of a sftp deployment, we could get the console output from the commands that can be run after upload.

i.e. upload a file and run a remote test suite on it.

Suspect we can do that with a combination of batch. sftp and app, but would be good if we could wrap it all up cleanly.

Thoughts ?

**Comment by mkloubert** [comment]
@enly1 

Sorry for the late answer.

**I have released a new version 9.30.0 which is now able to output the results of a command that is executed on a SFTP server.**

An example:

```json
{
    "deploy": {
        "targets": [
            {
                "type": "sftp",

                // ...

                "noCommandOutput": false,

                "connected": [
                    {
                        "command": "whoami",
                        "writeOutputTo": "current_bash_user",
                        "executeBeforeWriteOutputTo": "$others['current_bash_user'].trim()"
                    }
                ],

                "uploaded": [
                    {
                        "command": "npm test",
                        "verbose": true
                    },

                    {
                        "command": "echo \"You have uploaded as ${current_bash_user}\"",
                        "verbose": true
                    }
                ]
            }
        ]
    }
}
```

[Here](https://github.com/mkloubert/vs-deploy/wiki/target_sftp#execute-commands-on-server) the updated wiki article, which describes the new feature in detail.

**Comment by enly1** [comment]
Thanks, will give it a try. Very well timed. Thanks as always.
